config/default.json

```diff
@@ -1,5 +1,6 @@
 {
   "providerUrl": "http://localhost:8545",
-  "network": "kovan",
-  "gasLimit": 6500000
-}
+  "network": "ganache",
+  "gasLimit": 6015000,
```

Comment:
this number should come from env var

Reply:
well it can/does, see the info in the README and lib/config.js

Reply:
that ok by you? or do you want to not have any default gas and force someone to pass it in?

Reply:
yea.it is ok to have a default value with an option to override it .